.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 6
+- node
 - 4
```

Comment:
Since v4 and v6 are actually both LTS right now, I'd prefer to have `4` + `6` + `node`.
And actually, I'd prefer to have `7` instead of `node`: it makes it clearer from the job list without having to look for the version in the console output (see image below), and it won't silently go to another major version when someone opens a PR right after it gets released. I'm fine having to bump the version in the file manually once in a while.

<img width="346" alt="screen shot 2016-11-19 at 22 02 41" src="https://cloud.githubusercontent.com/assets/113730/20460038/2a005f72-aea4-11e6-8e26-f92f1e9a9f04.png">